CHANGELOG.next.asciidoc

```diff
@@ -72,6 +72,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix marshaling of ms-since-epoch values in `elasticsearch/cluster_stats` metricset. {pull}14378[14378]
 - Log bulk failures from bulk API requests to monitoring cluster. {issue}14303[14303] {pull}14356[14356]
 - Fixed bug with `elasticsearch/cluster_stats` metricset not recording license ID in the correct field. {pull}14592[14592]
+- Fix rds Closing handler after vmetricset from reporting same values for different instances. {pull}14702[14702]
```

Comment:
I think sth went wrong here! :) Could you check?

Reply:
Good call! Thanks @ChrsMark! Now sure what happened but it's fixed now 😄